CHANGELOG.md

```diff
@@ -1,4 +1,71 @@
+* networksecurity: added `locations` field to `google_network_security_mirroring_deployment_group` resource ([#9607](https://github.com/hashicorp/terraform-provider-google-beta/pull/9607))
+* networksecurity: added `locations` field to `google_network_security_mirroring_endpoint_group_association` resource ([#9603](https://github.com/hashicorp/terraform-provider-google-beta/pull/9603))
+* parametermanager: added `kms_key_version` field to `google_parameter_manager_parameter_version` resource and datasource ([#9642](https://github.com/hashicorp/terraform-provider-google-beta/pull/9642))
+* parametermanager: added `kms_key` field to `google_parameter_manager_parameter` resource and `google_parameter_manager_parameters` datasource ([#9642](https://github.com/hashicorp/terraform-provider-google-beta/pull/9642))
+* provider: added `external_credentials` block in `provider` ([#9658](https://github.com/hashicorp/terraform-provider-google-beta/pull/9658))
+* redis: added `automated_backup_config` field to `google_redis_cluster` resource ([#9682](https://github.com/hashicorp/terraform-provider-google-beta/pull/9682))
+* storage: added `content_base64` field in `google_storage_bucket_object_content` datasource ([#9638](https://github.com/hashicorp/terraform-provider-google-beta/pull/9638))
+
+BUG FIXES:
+* alloydb: added a mutex to `google_alloydb_cluster` to prevent conflicts among multiple cluster operations ([#9604](https://github.com/hashicorp/terraform-provider-google-beta/pull/9604))
+* artifactregistry: fixed type assertion panic in `google_artifact_registry_repository` resource ([#9672](https://github.com/hashicorp/terraform-provider-google-beta/pull/9672))
+* bigtable: fixed `automated_backup_policy` field for `google_bigtable_table` resource ([#9627](https://github.com/hashicorp/terraform-provider-google-beta/pull/9627))
+* cloudrunv2: fixed the diffs for unchanged `template.template.containers.env` in `google_cloud_run_v2_job` resource ([#9681](https://github.com/hashicorp/terraform-provider-google-beta/pull/9681))
+* compute: fixed a regression in `google_compute_subnetwork` where setting `log_config` would not enable flow logs without `enable_flow_logs` also being set to true. To enable or disable flow logs, please use `log_config`. `enable_flow_logs` is now deprecated and will be removed in the next major release. ([#9679](https://github.com/hashicorp/terraform-provider-google-beta/pull/9679))
+* compute: fixed unable to update the `preview` field for `google_compute_region_security_policy_rule` resource ([#9614](https://github.com/hashicorp/terraform-provider-google-beta/pull/9614))
+* compute: fixed unable to update the `preview` field for `google_compute_security_policy_rule` resource ([#9614](https://github.com/hashicorp/terraform-provider-google-beta/pull/9614))
+* orgpolicy: fix permadiff in `google_org_policy_policy` when multiple rules are present ([#9611](https://github.com/hashicorp/terraform-provider-google-beta/pull/9611))
```

Comment:
```suggestion
* orgpolicy: fixed permadiff in `google_org_policy_policy` when multiple rules are present ([#9611](https://github.com/hashicorp/terraform-provider-google-beta/pull/9611))
```

Reply:
Oh whoops- I had clicked merge before seeing this (it turns out we need to bypass anyways, since we don't have appropriate CODEOWNERS ourselves)

Reply:
Sorry, I assumed this had already been reviewed during release, so I approved it without a detailed review.